controller.php

```diff
@@ -275,6 +275,14 @@
                 echo '{"status":"error","message":"Missing parameter!"}';
             }
             break;
+        
+        case 'fetch':
+            if (isset($_GET['path']) && isset($_GET['remote'])) {
+                echo $git->fetch(getWorkspacePath($_GET['path'], $_GET['remote']));
```

Comment:
Please correct the parentheses:
`echo $git->fetch(getWorkspacePath($_GET['path']), $_GET['remote']);`